tests/models_tests/test_mlp.py

```diff
@@ -18,17 +16,14 @@ def model():
 
 @pytest.fixture
 def data():
-    atom_data = numpy.random.randint(
-        0, high=MAX_ATOMIC_NUM, size=(batch_size, atom_size)
-    ).astype(numpy.float32)
-    y_grad = numpy.random.uniform(
-        -1, 1, (batch_size, out_dim)).astype(numpy.float32)
-    return atom_data, y_grad
+    hidden = numpy.random.rand(1, hidden_dim).astype(numpy.float32)
+    y_grad = numpy.random.uniform(-1, 1, (1, out_dim)).astype(numpy.float32)
```

Comment:
Could you use `batch_size = 2` instead of hard coding 1 for the axis=0 of hidden and y_grad?

Reply:
Sorry about that. It should be fixed now.